ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -712,6 +712,27 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure Guest Access to Shared Folders Is Disabled (MDM Required)
```

Comment:
Now that we are using the plist method you can remove the MDM required suffix.

Reply:
right